flexget/components/sites/sites/filelist_api.py

```diff
@@ -0,0 +1,180 @@
+        'additionalProperties': False,
+    }
+
+    def get(self, url, params):
+
+        try:
+            response = requests.get(url, params=params, raise_status=False)
+        except RequestException as e:
+            raise plugin.PluginError(f'FileList request failed badly! {e}')
+
+        if response.status_code in self.error_codes.keys():
+            # get error message if we have one, else use generic
+            error_message = response.json().get('error', self.error_codes[response.status_code])
+
+            raise plugin.PluginError(
+                f'FileList request failed; err code: {response.status_code}; err msg: `{error_message}`'
+            )
+
+        return response
+
+    def dump_entry(self, entry):
+        entry_items = {key: value for key, value in entry.items()}
+        logger.debug(f'entry_items={entry_items}')
+
+    @plugin.internet(logger)
+    def search(self, task, entry, config):
+        """
+            Search for entries on FileList
+        """
+        entries = list()
+
+        # TODO remove later
+        self.dump_entry(entry)
+
+        # mandatory params
+        params = {'username': config['username'], 'passkey': config['passkey'], 'action': 'search-torrents'}
+
+        # category
+        if config.get('category'):
+            params['category'] = (
+                ','.join([self.valid_categories[cat] for cat in config['category']])
+                if isinstance(config.get('category'), list)
+                else self.valid_categories[config.get('category')]
+            )
+
+        # extras: internal release, moderated torrent, freeleech
+        for extra_name, extra_value in self.valid_extras.items():
+            if config.get(extra_name):
+                params[extra_name] = extra_value
+
+        # set season/episode if series
+        if entry.get('series_episode'):
+            params['episode'] = entry.get('series_episode')
+        if entry.get('series_season'):
+            params['season'] = entry.get('series_season')
+
+        for search_title in entry.get('search_strings', [entry.get('title')]):
+
+            if entry.get('imdb_id'):
+                params['type'] = 'imdb'
+                params['query'] = entry.get('imdb_id')
+                params['name'] = search_title
+            else:
+                params['type'] = 'name'
+                params['query'] = search_title
+
+            # TODO remove later
+            logger.debug(f'get params: {params}')
+
+            # get them results
+            try:
+                response = self.get(self.api_url, params)
+                # TODO remove later
+                logger.debug(f'Requesting: {response.url}')
+            except RequestException as e:
+                raise plugin.PluginError(f'FileList request failed badly! {e}')
+
+            results = response.json()
+            if not results:
+                logger.info(f'No torrent found on Filelist for `{search_title}`')
```

Comment:
Dont use fstrings in logger. See how it's used in other loggers. Also, I'm not sure this should be info level, verbose should be sufficient

Reply:
I will change the level to verbose and I will look into fstrings and logger.